domain/src/test/java/com/example/domain/GetLaunchesUseCaseTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.domain
+
+import com.apollographql.apollo3.api.ApolloResponse
+import com.example.spacexdemoapp.api.LaunchMapper
+import junit.framework.TestCase
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Answers
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+import spacexdemoapp.GetLaunchesQuery
+import java.util.UUID
+
+@RunWith(MockitoJUnitRunner::class)
+class GetLaunchesUseCaseTest : TestCase() {
+
+    @get:Rule
+    val instantExecutorRule = androidx.arch.core.executor.testing.InstantTaskExecutorRule()
+
+    @Test
+    operator fun invoke() {
```

Comment:
It is a testing request, response, and mapping which are a part of data module. So, now it should be in the data module as a part of repository testing.